### PR TITLE
Add web client example

### PR DESCRIPTION
**Pull request by rom1504**
mostly based on https://github.com/Heath123/mineflayer-web-test
@Heath123
@Karang

### CODE REVIEW

File: examples/web_client/package.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        "crypto-browserify": "^3.12.0",
+        "events": "^3.2.0",
+        "http-browserify": "^1.7.0",
+        "https-browserify": "^1.0.0",
+        "memfs": "^3.2.0",
+        "mineflayer": "^2.39.2",
+        "net-browserify": "^0.2.4",
+        "os-browserify": "^0.3.0",
+        "path-browserify": "^1.0.1",
+        "prismarine-viewer": "^1.13.0",
```
Comment (Karang) [inline examples/web_client/package.json:28]: Use the relative path, otherwise it reinstall everything.

File: examples/web_client/package.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
+{
+    "name": "prismarine-viewer-standalone",
```
Comment (Karang) [inline examples/web_client/package.json:2]: Should we change the name ?

**Review by Karang — COMMENTED** [review]
LGTM, 2 minor comments

File: examples/web_client/package.json (hunk 1 above)
Comment (rom1504) [reply examples/web_client/package.json:28]: indeed

File: examples/web_client/package.json (hunk 2 above)
Comment (rom1504) [reply examples/web_client/package.json]: yeah done

**Review by Karang — APPROVED** [review]

**Pull request merged**